src/components/tests/Episode.test.js

```diff
@@ -1,10 +1,48 @@
 import React from 'react';
-import { render, fireEvent, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import Episode from './../Episode';
 
-test("renders without error", () => { });
+const testEpisode = {
+   id: 1,
+   name: "",
+   image: "http://static.tvmaze.com/uploads/images/medium_landscape/67/168918.jpg",
+   season: 1,
+   number: 1,
+   summary: "This is a summary of the episode.",
+   runtime: 1
+}
 
-test("renders the summary test passed as prop", () => { });
+const testEpisodeWithoutImage = {
+   id: 2,
+   name: "",
+   image: null,
+   season: 2,
+   number: 3,
+   summary: "This is a summary of the episode.",
+   runtime: 43
+}
 
-test("renders default image when image is not defined", () => { });
+
+test("renders without error", () => {
+   render(<Episode episode={testEpisode} />)
+});
+
+test("renders the summary test passed as prop", () => {
+   render(<Episode episode={testEpisode} />)
+
+   const summary = screen.getByText(/this is a summary of the episode./i)
+
+   expect(summary).toBeInTheDocument()
+   expect(summary).not.toBeNull()
+   expect(summary).toBeDefined()
```

Comment:
You don't need both of these. If summary is defined then it will automatically not be null and vice versa. You don't need both expect statements here.